CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 30.0.0
+
+- Removed: rules deprecated in `stylelint` `15.0.0`.
```

Comment:
[trivial] It also may be good to use the `<name>@<version>` format of [`npm install`](https://docs.npmjs.com/cli/v9/commands/npm-install):

```suggestion
- Removed: rules deprecated in `stylelint@15.0.0`.
```